src/main/resources/default-lintConfig.xml

```diff
@@ -136,4 +136,10 @@
 
     <!-- Enable checking for unused ids -->
     <issue id="UnusedIds" severity="info"/>
+
+    <!-- Suppress illegal package error, see https://github.com/square/okio/issues/58 -->
+    <issue id="InvalidPackage">
+        <ignore regexp="okio-.*.jar"/>
+        <ignore regexp="retrofit-.*.jar"/>
```

Comment:
Added requery for now, and open to muting the larger issue later, but didn't want to make that decision at this moment.